ocs_ci/cleanup/aws/defaults.py

```diff
@@ -6,6 +6,6 @@
     'jnk': 36, 'dnd': 'never', 'lr1': 24, 'lr2': 48, 'lr3': 72, 'lr4': 96,
     'lr5': 120
 }
-MINIMUM_CLUSTER_RUNNING_TIME = 10
+MINIMUM_CLUSTER_RUNNING_TIME = 120
```

Comment:
Why?

Reply:
10 hours is way to less to do meaningful testing

Reply:
12 hours sound like a nice to have fix.
If you are working more than that - why? (-:
if not and this is just for ease of use - this is the intent to clean the clusters

Reply:
It takes around ~30 mins to setup, I wont be awake in night to test cluster :) but having the cluster running for long time is needed for many tests rather than setup new everytime, the age of the cluster is also important overtime.

Reply:
@vasukulkarni this should not be changed. It used for determine what is the minimum running time in hours any cluster is allowed to have. With 120 it means that aws-cleanup will not work as it passed 12 hours as the allowed running time for any cluster that doesn't have the exceptions in its name (like jnk or lr)

Reply:
I think i am missing how the code works, but I am closing this anyway now and going to use the prefix for now